src/block-variations/core/gallery/index.js

```diff
@@ -46,6 +47,17 @@ export const BLOCK_VARIATION_GALLERY_COLLAGE = {
 	title: __( 'Collage (CoBlocks)', 'coblocks' ),
 };
 
+export const BLOCK_VARIATION_GALLERY_OFFSET = {
+	attributes: {
+		className: 'is-style-offset',
```

Comment:
Looks like the `offset` style is missing. Let's add it to the `src/block-styles/core/gallery` directory.